### ISSUE TITLE
Convert trained yolov3 Darknet-53 custom model to tensorflow model

### ISSUE THREAD

**Issue by rohit167** [microsoft/MMdnn#832]
Platform: ubuntu 16.04
Python version: 3.6
Source framework:  mxnet (gluon-cv)
Destination framework: Tensorflow
Model Type: Object detection
Pre-trained model path: [https://gluon-cv.mxnet.io/build/examples_detection/demo_yolo.html#sphx-glr-build-examples-detection-demo-yolo-py](url)

I have trained a custom yolov3 Darknet-53 model (yolo3_darknet53_custom.params) using gluon-cv (mxnet). I need to convert the yolo3_darknet53_custom.params (mxnet) model to yolo3_darknet53_custom.pb (tensorflow)

Also, I see https://pypi.org/project/mmdnn/ object detection is under on-going Models.

Queries:
Does mmdnn supports yolo models (object detection) conversion trained using gluon-cv (mxnet) in general?
Is there any way or work around by which I can convert yolo models?

Any leads would be great!
Thank you

**Comment by XiaoXYe** [comment]
Hi @rohit167 , we haven't test MMdnn with yolo models, so there might be some unsupported ops and accuracy loss.
